applications/matter_weather_station/README.rst

```diff
@@ -166,6 +166,14 @@ The application comes with the following overlays:
   See `Generating factory data`_ to learn how to put factory data into device's storage.
   To learn more about factory data, read the :doc:`matter:nrfconnect_factory_data_configuration` page in the Matter documentation.
 
+  This overlay requires providing of the :file:`pm_static_factory_data.yml` Partition Manager static configuration file.
+
+  To build the example with the factory data support, run the following command:
+
+  .. code-block:: console
+
+      west build -b thingy53_nrf5340_cpuapp -- -OVERLAY_CONFIG=overlay-factory_data.conf -DPM_STATIC_YML_FILE=pm_static_factory_data.yml
```

Comment:
```suggestion
      west build -b thingy53_nrf5340_cpuapp -- --DOVERLAY_CONFIG=overlay-factory_data.conf -DPM_STATIC_YML_FILE=pm_static_factory_data.yml
```
I'm being told though that `OVERLAY_CONFIG` is deprecated and `EXTRA_CONF_FILE` should be used instead. Can you use `EXTRA_CONF_FILE`, please?

Reply:
Done, I've changed it also in Matter Light Bulb sample.